homework/12_B/homework1/krugove.py

```diff
@@ -0,0 +1,20 @@
+from math import sqrt
```

Comment:
It is a good practice to use english i.e. instead of `krugove.py` -> `circles.py`